### ISSUE TITLE
slimScroll only working after resizing browser window

### ISSUE THREAD

**Issue by runnee** [alvarotrigo/fullPage.js#766]
Refer to: https://a002989df921a8b94c081c0b6c4389dccb976658.googledrive.com/host/0B-vJE2R2dId6XzJ4cE5tbm00RU0/

The website alignment is doing fine until I added slimScroll.

Since then, everything becomes weird. I found out that all the section takes up the height of the section with the largest value. The scrollBar on the right is also not working for each section as seen on first load. But when I resize the window and maximize it again, everything is working find, including the scrollBar loaded.

I tried using it on window on load but it is the same.

Anyone can help on this?

**Comment by Reeslo** [comment]
I have the same isue after ajax content loading sometime it works and sometime I have to resize the windows.

https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.7.4/vendors/jquery.slimscroll.min.js
https://cdnjs.cloudflare.com/ajax/libs/fullPage.js/2.7.4/jquery.fullPage.min.js

**Comment by alvarotrigo** [comment]
> I have the same isue after ajax content loading sometime it works and sometime I have to resize the windows.

You need to use `reBuild`. That's not an issue.

**Comment by Reeslo** [comment]
I tried a lot of process :

Ajax call
Change main container element
destroy then rebuild 
then call fullpage

Ajax call 
Change main container
rebuild then destroy
then call full-page

I tried same only with rebuild it destroy.

Yesterday after my message I tried other process
Ajax call
destroy
changing content
re-destroy
rebuild
For the moment I don't see the problem

Idk I that is the good process or not. But sometime slimscroll appear, other time after ajax reload slimscroll are not visible on all section who had big content, but only on the 1st section (soz for my English).
